markdown/faq.md

```diff
@@ -38,12 +38,12 @@ No. We don’t provide any guarantees of privacy on the public cluster, so priva
 ### can anybody build a Binder from my repository?
 
 Yes. We do not require authentication when building from a repository. This makes it
-simple to get started, but cannot prevent others from rebuilding an existing Binder. However, as of the 1.0.0 version Binders are specified deterministically from repositoey contents. So repeated rebuilds of the same repository will be identical, and changes to the environment will require changing the repository itself.
+simple to get started, but cannot prevent others from rebuilding an existing Binder. However, as of the 1.0.0 version Binders are specified deterministically from repository contents. So repeated rebuilds of the same repository will be identical, and changes to the environment will require changing the repository itself.
 
 ### is there any extra information about the project?
 
 We recommend this great [blog post](http://ivory.idyll.org/blog/2016-mybinder.html) by C. Titus Brown.
 
 ### how can I get involved?
 
-We welcome new contributors! Check out the "devs" section of this documentation site, open an issue or pull request on one of the project repositories on [GitHub](https://github.com/binder-project), or come talk to us in the [chatroom](https://gitter.im/binder-project/binder).
\ No newline at end of file
+We welcome new contributors! Check out the "devs" section of this documentation site, open an issue or pull request on one of the project repositories on [GitHub](https://github.com/binder-project), or come talk to us in the [chatroom](https://gitter.im/binder-project/binder).
```

Comment:
hmm not sure what happened here, the two lines look identical to me / I didn't make any changes.